langlib/lang.decimal/src/main/ballerina/decimal.bal

```diff
@@ -27,6 +37,16 @@ public isolated function sum(decimal... xs) returns decimal = @java:Method {
 
 # Returns the maximum of one or more decimal values.
 #
+# ```ballerina
+# decimal:max(1.2, 12.3, 3.4) ⇒ 12.3
+#
+# decimal[] marks = [70.3, 80.5, 98.1, 92.3];
+# decimal:max(30.5, ...marks) ⇒ 98.1
```

Comment:
Can use the suggestion given at: https://github.com/ballerina-platform/ballerina-lang/pull/38917#discussion_r1052975131

Reply:
Updated. Please check